ext/standard/type.c

```diff
@@ -434,6 +450,135 @@ PHP_FUNCTION(is_callable)
 }
 /* }}} */
 
+#define RETURN_TYPE_NAME(t) { RETURN_STRINGL(t, sizeof(t) - 1) }
```

Comment:
That would only apply to the extended mode, right? The constants are already constant and the `sizeof` will be replaced with the actual number anyways during compilation. Just asking to gain a better understanding of C.

[Reading up](http://jpauli.github.io/2015/09/18/php-string-management.html) on interned strings now.